bin/cli.js

```diff
@@ -65,18 +63,13 @@ const cliFlags = yargs
     return true;
   }).argv;
 
-//Get all explicitly terminal set options, does not include url because url has no default option
-const flags = Object.keys(yargs.getOptions().default).reduce((accum, prop) => {
-  if (cliFlags[prop] !== yargs.getOptions().default[prop])
-    accum[prop] = cliFlags[prop];
-  else if (cliFlags.config && cliFlags.config.flags && cliFlags.config.flags[prop])
-    accum[prop] = cliFlags.config.flags[prop];
-  else accum[prop] = yargs.getOptions().default[prop];
-  return accum;
-}, {});
 
-//Merge options from all sources. Order indicates precedence (last => most important)
-let options = Object.assign({}, cliFlags.config, { flags: flags });
+const config = cliFlags.config ? getConfigFromFile(cliFlags.config) : {};
+
+// For the config flags and URL, we'll allow CLI to override what's set in the config Object
```

Comment:
I really don't need this logic right now (mix up config and cli flags) but if @pedro93 is needed, maybe it makes sense to have it.

From another side. We always can add it in for e.g. 2.1 to not block 2.0 release, it's also better to add API then drop it (if drop we will have to make 3.0 release). Maybe after 2.0 more users request this so it will be useful.